buildspec.test.yml

```diff
@@ -0,0 +1,50 @@
+version: 0.2
+
+env:
+  parameter-store:
+    CYPRESS_baseurl: CognitoFrontendApplicationUrl
+
+phases:
+  install:
+    runtime-versions:
+      nodejs: 14
+      python: 3.x
+    commands:
+      - echo "install cypress dependencies..."
+      #- apt-get update
+      #- apt-get install -y libgtk2.0-0 libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 xvfb jq
+  pre_build:
+    commands:
+      - wget https://github.com/BIBSYSDEV/NVA-end-to-end-testing/archive/refs/heads/test-deploy.zip
+      - unzip test-deploy.zip
```

Comment:
Create task for improvement of checking if changes exist

Reply:
https://unit.atlassian.net/browse/NP-3067